ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -3005,20 +3005,18 @@ periodics:
         secretName: ci-pull-credentials
 - agent: kubernetes
   decorate: true
-  extra_refs:
-  - base_ref: release-4.2
-    org: openshift
-    repo: openshift-ansible
-    workdir: true
-  interval: 3h
+  interval: 12h
   labels:
+    ci.openshift.io/release-type: informing
+    job-release: "4.2"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: release-openshift-openshift-ansible-e2e-aws-scaleup-rhel7-4.2
+  name: release-openshift-ocp-installer-e2e-aws-rhel7-workers-4.2
```

Comment:
Side note; new terminology "rhel7-workers" makes SO much more sense to me than "scaleup".